requirements.txt

```diff
@@ -1,3 +1,4 @@
-fondant==0.9.0
+fondant[component,aws,azure,gcp,docker]==0.10.1
 notebook==7.0.6
 weaviate-client==3.25.3
+langchain==0.0.329
```

Comment:
Do we need this here if it's in the `extra_requires` of the component?

Reply:
Yep, we do have langchain as extra requirement. But at the moment we are not installing the extra requirements in the local environment. When I am in the notebook, and want to test the class implementation, I will get an exception that the dependency is missing. I think this is something we should takle in [#830](https://github.com/ml6team/fondant/issues/830)

Reply:
But we are not doing the same for eg. Ragas?

Since we are not actually executing the components in the notebook at this moment, I would leave them out of the requirements.txt.